docs_user/modules/openstack-backend_services_deployment.adoc

```diff
@@ -36,14 +42,12 @@ you've run `make crc_storage`.
 * Set the desired admin password for the podified deployment. This can
 be the original deployment's admin password or something else.
 +
-[,bash]
```

Comment:
When we remove this, the block renders with white background, while the blocks which have this "header" render with black background. Not an issue if it's not an issue downstream, just making sure we know about it. Here's the preview:

https://logserver.rdoproject.org/64/264/dccf1357734825bc1672d7160e12a2926772cb95/github-check/adoption-docs-preview/29a7a68/docs_build/docs_build/adoption-user/index-upstream.html